org-eclipse-che-jdt-ls-extension/jdt.ls.extension.core/src/main/java/org/eclipse/che/jdt/ls/extension/core/internal/JavaModelUtil.java

```diff
@@ -85,6 +94,55 @@ static SymbolKind mapKind(IJavaElement element) {
     return DocumentSymbolHandler.mapKind(element);
   }
 
+  /**
```

Comment:
Why move this here? It's only ever used in FindImplementersHandler.

Reply:
It will be useful to solve:  https://github.com/eclipse/che/issues/10874